openesdh-repo/src/main/java/dk/openesdh/repo/services/audit/OpenESDHAuditQueryCallBack.java

```diff
@@ -32,44 +40,175 @@ public JSONArray getResult() {
         return result;
     }
 
+    @Override
+    public boolean valuesRequired() {
+        return true;
+    }
+
+    @Override
+    public boolean handleAuditEntry(Long entryId, String applicationName, String user, long time, Map<String, Serializable> values) {
```

Comment:
Seems like this method needs a refactoring according to the Strategy pattern or behavior parameterization.